VMS/Pages/ActivityInfoPage/InfoPage.razor.scss

```diff
@@ -0,0 +1,163 @@
+﻿@import '../../Resources/scss/common/_variables';
+
+.TopPage {
+    .Return-text {
+        text-decoration: none;
+        font-family: Montserrat;
```

Comment:
DO NOT set font-family, please remove this all other places.

Reply:
done